covid_model/sql/hospitalized_county_subset.sql

```diff
@@ -0,0 +1,7 @@
+select
+    measure_date
+    , sum(hospitalized) as currently_hospitalized
+from cophs.hospitalized_by_county
+where county_id in unnest(%(county_ids)s)
+group by 1
+order by 1
```

Comment:
Does this work? you're not selecting county_id but using it in the where clause. Maybe that's fine, I'm not an SQL expert.